test/Microsoft.ML.CpuMath.UnitTests/UnitTests.cs

```diff
@@ -611,7 +611,7 @@ public void SumSqUTest(string mode, string test, Dictionary<string, string> envi
                 }
 
                 var actual = CpuMathUtils.SumSq(src);
-                Assert.Equal(expected, actual, 2);
+                Assert.Equal((double)expected, (double)actual, 2);
```

Comment:
just wondering what is wrong in comparing as `float`. or `Equal` doesn't have this overload?

Reply:
I mentioned the [xunit issue](https://github.com/xunit/xunit/issues/2393) in https://github.com/dotnet/machinelearning/pull/6354/commits/78fe6eeed6cc16c306594279f0fa9f000247cbdd that has the details.
We had to make the same workarounds in dotnet/runtime.  Xunit could avoid this break by adding a `float, float, int` overload.